docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-configuring-and-running-an-application-analysis_{context}"]
+= Configuring and running an application analysis
+
+You can analyze more than one application at a time against more than one transformation target in the same analysis.
+
+.Procedure
+
+. In *Development* view, click *Application inventory*.
+. Click the *Analysis* tab.
+. Select the applications you want to analyze.
++
+// ![](/Tackle2/AppAssessAnalyze/SelectManageCred.png)
+
+. Review the credentials assigned to the application.
+. Click *Analyze*.
++
+// ![](/Tackle2/AppAssessAnalyze/AnalysisMode.png)
+
+. Select the *Analysis mode* from the list. Valid options are:
+* Binary.
+* Source code.
+* Source code and dependencies.
+* Upload a local binary. This option only appears if you are analyzing a single aplication.
+
+. If you choose *Upload a local binary*, a window opens and you are prompted to *Upload a local binary*. Either drag and drop a file into the area provided or click *Upload* and then select the file to upload.
+
+. Click *Next*.
+. Select one or more transformation targets:
```

Comment:
I think you forgot to change this line to **Select one or more target options in focus for the analysis**.

Reply:
@kpunwatk I actually do not want the command to be identical to the UI -- we don't usually write an entire instruction in bold, but I did change the command to fit the UI better.

Reply:
Oh, Got it, No Problem !
Thanks :+1:

Reply:
@kpunwatk So is the PR OK or is there anything else left?